### ISSUE TITLE
Encuesta de sueldos España , 12, 14, o 15 pagas anuales.

### ISSUE THREAD

**Issue by mabeett** [sysarmy/disneyland#154]
**La sugerencia se relaciona con un problema? **

Sí, se podría distorsionar la cooperación de sueldos si se reciben 12, 14 o 15 pagas anuales.

```
El número que figura en tu recibo de sueldo antes de impuestos, cargas sociales, etc.
```

En España para el trabajo en relación de dependencia se pagan dos sueldos aguinaldos (Argentina dos medios) aunque también se pueden prorratear, además para más alegría y diversidad algunos empleadores no pagan el año en 12 cuotas sino en 15.

**Vi algo que quiero que vean si pueden hacer.**

Tal vez se puede poner algo tipo "multiplicá por N/12 si recibis N pagos anuañes a fin de que el número quede parejo para todos", O preguntar la cantidad de pagas anuales.

No sé, al final del día acá la spec de salario anual bruto (antes de rentss/ganancias y seguridad Social y alguna otra feta).

**Lo puedo hacer yo? o puedo colaborar?**

Más que estos comentarios no sé decir.

**Comment by godlike64** [comment]
El tema de salario mensual vs anual, en Argentina se suele hablar de sueldo mensual (por más que en cualquier empresa en realidad se está asignando un sueldo anual a la persona), mientras que en muchos otros lados (al menos en Europa/US) es siempre sueldo anual. La diferencia entre sueldo bruto/neto no dice mucho, ya que el neto va a depender de cada persona (por ejemplo, lo que declare para deducir de Impuesto Ganancias), mientras que el bruto sí da información sobre el puesto, que es lo que interesa.

Quizás (no para esta encuesta, porque ya está empezada) lo ideal sería sacar ambos campos de sueldo mensual bruto/neto y reemplazarlos por:

- Sueldo anual bruto
- Sueldo mensual bruto (+ otro campo con "Cantidad de pagas"?)

Tiro esto porque no conozco cómo es en cada país. En Argentina el sueldo anual ~= sueldo mensual x 13 por ejemplo, en España depende de si 12 o 14 pagas, etc. Lo ideal sería converger en sueldo anual pero no se si todo el mundo va a conocer su sueldo anual.

**Comment by Nachichuri** [comment]
> Lo ideal sería converger en sueldo anual pero no se si todo el mundo va a conocer su sueldo anual.

Coincido con este ultimo comentario de godlike, creo que cuanto más sencilla sea la encuesta más fácil es conseguir más volumen de respuestas, si culturalmente no estamos acostumbrados a calcular sueldos anuales quizás se agrega un step mas de cálculo que genera que la gente abandone la encuesta por la mitad...

Otra posible complejidad para el cálculo es que preguntar sueldos anuales, cuando en muchas empresas se hacen ajustes trimestrales por inflación, te deja con valores nominales que quizás no son posta. También creo que la gente tiene más fresco el número del sueldo mensual y que por ahí calcularía un estimativo anual tirando fruta sólo para completar la pregunta, lo que puede terminar distorsionando los valores reales.

> Tal vez se puede poner algo tipo "multiplicá por N/12 si recibís N pagos anuales" a fin de que el número quede parejo para todos, O preguntar la cantidad de pagas anuales.

De todas maneras esta idea me sigue pareciendo copada, habría que buscar una forma de implementarlo que no sume carga operativa al momento de completar la encuesta.

**Comment by godlike64** [comment]
> Coincido con este ultimo comentario de godlike, creo que cuanto más sencilla sea la encuesta más fácil es conseguir más volumen de respuestas, si culturalmente no estamos acostumbrados a calcular sueldos anuales quizás se agrega un step mas de cálculo que genera que la gente abandone la encuesta por la mitad...
> 
> Otra posible complejidad para el cálculo es que preguntar sueldos anuales, cuando en muchas empresas se hacen ajustes trimestrales por inflación, te deja con valores nominales que quizás no son posta. También creo que la gente tiene más fresco el número del sueldo mensual y que por ahí calcularía un estimativo anual tirando fruta sólo para completar la pregunta, lo que puede terminar distorsionando los valores reales. Es el mismo principio que en España para la gente que cobra en 14 pagas: no cobran 2 sueldos extras al año, sino que el sueldo anual se divide por 14 y eso es lo que se paga cada mes (o en cada "paga"), en vez de dividirse por 12 y pagar eso cada mes.

Sí, es verdad que es un cálculo más, pero en Argentina estamos mal acostumbrados con lo del sueldo mensual, por así decirlo. La realidad es que los sueldos se calculan anualmente, y el aguinaldo no es "un sueldo extra" sino que las empresas ya lo tienen considerado para cada empleado en la contabilidad anual. Es simplemente plata que se te retiene y se te entrega cada N meses.

Por otra parte, para los que laburan en empresas grandes o de afuera que tienen alguna solución de software de HR, deberían poder ver ahí su sueldo anual (que dividiéndolo por 13 en Argentina les va a dar el mensual bruto, por ejemplo).

Lo de los ajustes trimestrales es verdad, pero aplica de la misma manera para los sueldos mensuales (y es uno de los motivos de hacer la encuesta 2 veces al año). Sabemos que es una foto que se saca cada ~6 meses, que probablemente al par de meses ya va a quedarse atrás (en términos de sueldos argentinos) pero no hay mucho que se pueda hacer al respecto.

**Comment by mabeett** [comment]
Hola chicos,

Me movilizó e hizo ruido que para residentes en un mismo territorio podría haber la misma respuesta cuando los salarios resultasen efecto bien distintos.

Tal vez ese pueda ser un punto de partida:  en un país la pregunta de manera y en otro se responde de otra siendo en cada uno de ellos la más cómodo/usual/etc y luego el normalizar los datos para compararlo entre países quedará en The experts (TM).
